Assignment-5.py

```diff
@@ -0,0 +1,233 @@
+"""Importing Necessary Packages"""
+import random
+import pandas as pd
+from datetime import datetime
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+class MobileShowroom:
+    """Offline Store"""
+    def __init__(self,brand,model,search):
+        self.brand=brand
+        self.model=model
+        self.search=search
+        
+    def launch(self):
+        print("NEW LAUNCHES OF THE WEEK")
+        nl=["Samsung","Xiaomi","Apple"]
+        return f"{nl}"
+    
+    """Specifying the user requirements"""    
+    def requirements(self,**kwargs):
+        print("\nYOUR REQUIREMENTS")
+        req=[]
+        for key,value in kwargs.items():
+            req.append("{} : {}".format(key,value))
+        return f"{req}"
```

Comment:
f strings are used when you want to add values inside normal text.
eg : f"this is response {req}".

when you want to return only req. you can directly return it as "req".